docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+For example, if you would like to train a {classification} model that decides 
+whether an email is a spam or not, you need a labelled data set that contains 
+enough data points from each possible category to train the model. What counts 
+as "enough" depends on various factors like the complexity of the problem or 
+the {ml} solution you have chosen. There is no exact number that fits every 
+use case; deciding how much data is acceptable is rather a heuristic process 
+that might involve iterative trials.
+
+Before you train the model, consider preprocessing the data. In practice, the 
+type of preprocessing depends on the nature of the data set. Preprocessing can 
+include, but is not limited to, mitigating redundancy, reducing biases, applying 
+standards and/or conventions, data normalization, and so on.
+
+{regression-cap} and {classification} requires specifically structured source 
+data: a two dimensional tabular data structure. For this reason, you might need 
+to {ref}/transforms.html[{transform}] your data to create a {dataframe} which 
+can be used as the source for {dfanalytics} like {regression} and 
+{classification}.
+
+[[train-test-iterate]]
+== Train, test, iterate
+
+After your data is prepared and transformed into the right format, it is time to 
+train the model. Training is an iterative process — every iteration is followed 
+by an evaluation to see how the model performs. When you are satisfied with the 
+results, you are ready to deploy the model otherwise you may want to adjust the 
+training configuration or consider alternative ways to preprocess and represent 
+your data.
+
+The first step is defining the features – the relevant fields in the data set – 
+that will be used for training the model. By default, all the fields with 
+supported types are included in {regression} and {classification} automatically. 
+However, you can exclude irrelevant fields optionally from the process. Doing so 
+makes a large dataset more manageable, reducing the computing resources and time 
+required for training.
+
+Now define how to split your data into a training and a test set. The test set 
+won’t be used to train the model, but is used to evaluate how the model 
+performs. There is no optimal percentage that fits all use cases, it depends on 
+the amount of data and the time you have to train. For large data sets, you may 
+want to start with a low training percent to complete an end-to-end iteration in 
+a short time.
+
+During the training process, the training data that contains the ground truth 
+you want to learn from is fed through the learning algorithm. The model predicts 
+the value and compares it to the ground truth then the model is fine-tuned to 
+make the predictions more accurate.
+
+Once the model has been trained, it will be tested on how well it predicts for 
+previously unseen data. This test provides an estimate of a quantity known as 
```

Comment:
I found "an estimate of a quantity" confusing, so if possible it would be simpler to just omit that:
```suggestion
previously unseen data with
```